src/pages/Settings/RecoveryCodeConfirm.vue

```diff
@@ -69,6 +69,8 @@ export default {
 		copyToClipboard(string) {
 			try {
 				navigator.clipboard.writeText(string);
+				// Tip message letting user know code successfully copied to clipboard
+				this.$showTipMsg('Recovery code copied.', 'confirm');
```

Comment:
This should work fine but you don't need the `confirm` param as it's not a valid type. You can just leave it off for the default style and icon to show.